<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>558</width>
    <height>422</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="DEBUG_BOX">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>541</width>
      <height>361</height>
     </rect>
    </property>
    <property name="title">
     <string>DEBUG</string>
    </property>
    <widget class="QPushButton" name="Debug_Mode_Change">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>70</y>
       <width>201</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>DEBUG_MODE_CHANGE</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="BPM_SHOW">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>25</y>
       <width>301</width>
       <height>101</height>
      </rect>
     </property>
     <property name="title">
      <string>BPM</string>
     </property>
     <widget class="QLineEdit" name="ppg_lv_text">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="ecg_lv_text">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>60</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="ppg_lv">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>64</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>PPG_LV</string>
      </property>
     </widget>
     <widget class="QLabel" name="ecg_lv">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>64</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>ECG_LV</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bpm_set">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>20</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>SET</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="UDAS">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>245</y>
       <width>301</width>
       <height>101</height>
      </rect>
     </property>
     <property name="title">
      <string>UDAS</string>
     </property>
     <widget class="QPushButton" name="Pedal_ERR_Set">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>20</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>SET</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="Pedal_ERR">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>8</x>
        <y>45</y>
        <width>101</width>
        <height>19</height>
       </rect>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial Narrow</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="text">
       <string>Pedal_Err</string>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="CAM">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>135</y>
       <width>301</width>
       <height>101</height>
      </rect>
     </property>
     <property name="title">
      <string>CAM</string>
     </property>
     <widget class="QLineEdit" name="cam_lv_text">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>50</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="cam_lv">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>64</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>CAM_LV</string>
      </property>
     </widget>
     <widget class="QPushButton" name="cam_set">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>20</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>SET</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="ALL_SET">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>142</y>
       <width>200</width>
       <height>200</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>굴림</family>
       <pointsize>26</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ALL SET</string>
     </property>
    </widget>
    <widget class="QLabel" name="Debug_Mode_Text">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>40</y>
       <width>201</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Debug_Mode_Disabled</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>558</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
